src/h5web/providers/mock/MockProvider.tsx

```diff
@@ -18,18 +17,14 @@ function MockProvider(props: Props): ReactElement {
     <Provider
       api={{
         domain,
-        getMetadata: async () => mockMetadata,
         getEntity: async (path: string) => {
           if (path === slowOnPath) {
             await new Promise((resolve) => {
               setTimeout(resolve, 3000);
             });
           }
 
-          const entity = getEntityAtPath(mockMetadata, path);
-          assertDefined(entity);
-
-          return entity;
+          return findMockEntity(mockMetadata, path);
```

Comment:
Note that I moved `assertDefined` inside `findMockEntity`, which is why I renamed it from `get` to `find`.